docs/source/configuration/locking.md

```diff
@@ -0,0 +1,21 @@
+# Locking support
+
+Volto provide support for Plone's locking feature. You need to enable `plone.locking` behavior on your Dexterity Content-Type that comes available by default. You can do that in Plone's control panel or using the Generic Setup facility.
```

Comment:
Volto provides support for Plone's locking feature, but you need to enable the `plone.locking` behavior on your Dexterity content types first. You can do that in Plone's control panel or using the Generic Setup facilities.